File: LB/PROgram0050.c
/*

Q50) Write a PROgram Which Accept Range from user ans Display all number in Reversein between that range 

    int :- 23 30
    output :-  30 29 28 27 26 25 24 23

*/

#include<stdio.h>

void ReverseDsiplay(int iStart, int iEnd)
{
    int iCnt = 0;

    printf("Reverse order of Given Range is : ");

    for(iCnt = iEnd; iCnt >= iStart; iCnt--)
    {
        printf("%d  ",iCnt);    
    }

}

int main()
{
    int iValue1 , iValue2 = 0;
    int iRet = 0;

    printf("Enter the First number : ");
    scanf("%d",&iValue1);

    printf("Enter the Second number : ");
    scanf("%d",&iValue2);

    ReverseDsiplay(iValue1, iValue2);

    return 0;
}

File: PM/PROgram0050.py
'''

Q10) Write a PROgram that print all Odd Number till N

'''

def Odd(No):

    for i in range(1, No + 1):
        if(i % 2 != 0):
            print(i)

def main():
    Value = int(input("Enter the First Input : "))

    Odd(Value)

if __name__ == "__main__":
    main()